api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,30 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const xToken = Object.getOwnPropertyDescriptor(req.headers, 'x-access-token');
```

Comment:
This is a bit overkill, you can use the `req.headers` object like any other object:

```JAVASCRIPT
cont xToken = req.headers['x-access-token'];
```